libs/ledger-live-common/src/families/celo/datasets/celo.scanAccounts.1.ts

```diff
@@ -38,9 +38,19 @@ const dataset: CurrenciesData<Transaction> = {
   ],
   accounts: [
     {
-      // Ignoring the "balance is sum of ops" test since that
-      // doesn't seem to take into account staked tokens properly.
-      FIXME_tests: ["balance is sum of ops"],
+      FIXME_tests: [
+        // Ignoring the "balance is sum of ops" test since that
+        // doesn't seem to take into account staked tokens properly.
+        "balance is sum of ops",
+        // Ignoring the following two tests for this account because they can occasionally fail
+        // due to the unique Celo behavior where we take into account pending staking operations
+        // to calculate balances.
+        //
+        // These two tests are will be run in the next account's tests - the second account
+        // does not include staking operations, so they will succeed.
+        "can be run in parallel and all yield same results",
```

Comment:
I didn't see a report where this test was failing, does it show occasional failure?